python/engine/test/AlfalfaMeasure/measure.py

```diff
@@ -0,0 +1,39 @@
+import typing
+
+import openstudio
+
+
+class AlfalfaMeasure(openstudio.measure.ModelMeasure):
+    def name(self):
+        return "Alfalfa Measure"
+
+    def modeler_description(self):
+        return "The method attempts to build an alfalfa json in the measure"
+
+
+    def arguments(self, model: typing.Optional[openstudio.model.Model] = None):
+        args = openstudio.measure.OSArgumentVector()
+
+        return args
+
+    def run(
+        self,
+        model: openstudio.model.Model,
+        runner: openstudio.measure.OSRunner,
+        user_arguments: openstudio.measure.OSArgumentMap,
+    ):
+        """
+        define what happens when the measure is run
+        """
+
+        alfalfa = runner.alfalfa()
+        alfalfa.exposeConstant(10, "safe value")
+        alfalfa.exposeMeter("Facility:Electricity", "Facility Electricity")
+        alfalfa.exposeActuator("somehting", "another thing", "key", "example actuator")
+        alfalfa.exposeOutputVariable("Whole Building", "Facility Total Purchased Electricity Energy", "output variable")
```

Comment:
Is this a change? I seem to recall that these originally took ModelObject instances. I think when we @TShapinsky last spoke you were contemplating a change.

Reply:
A risk I can think of is that the final names might not be known at ModelMeasure time.